### PR TITLE
Running formatter on build_and_redeploy

### PR DESCRIPTION
**Pull request by andresilva91**
## Motivation

We might want to standardize formatting for these scripts. I use `shell-format` for shell scripts, Dockerfile, gitignore, etc

## Proposal

Formatting `build_and_redeploy.sh` with it

## Test Plan

CI

### CODE REVIEW

**Comment by andresilva91** [comment]
Current dependencies on/for this PR:
* main
  * **PR #1204** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1204" target="_blank">Graphite</a> 
    * **PR #1205** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1205" target="_blank">Graphite</a>  👈
      * **PR #1206** <a href="https://app.graphite.dev/github/pr/linera-io/linera-protocol/1206" target="_blank">Graphite</a> 

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/1205?utm_source=stack-comment).

File: kubernetes/linera-validator/build_and_redeploy.sh [hunk 1]
```diff
@@ -28,12 +34,15 @@ usage() {
...
+        -h | --help)
+            usage
+            exit 0
+            ;;
         --cloud) cloud_mode=1;;
-        --port-forward) port_forward=1;;
-        --no-build) do_build=;;
-        --clean) clean=1;;
-        --copy) copy=1;;
+        --port-forward) port_forward=1 ;;
```
Comment (Twey) [inline kubernetes/linera-validator/build_and_redeploy.sh:42]: I'm confused what happened here — it wants a space for line 42 but not line 41?  Why?

File: kubernetes/linera-validator/build_and_redeploy.sh [hunk 2]
```diff
@@ -96,15 +105,15 @@ else
...
-kubectl get svc;
+echo "Pods:"
+kubectl get pods
+echo -e "\nServices:"
+kubectl get svc
 
-docker rm linera-test-local;
-docker run -d --name linera-test-local "$docker_image" \
-    && docker cp linera-test-local:wallet.json /tmp/ \
-    && docker cp linera-test-local:linera.db /tmp/
```
Comment (Twey) [inline kubernetes/linera-validator/build_and_redeploy.sh:107]: TBH I think that sequences like these in scripts might be better served by `errexit` anyway.

**Comment by Twey** [comment]
The changes look largely sensible to me, though I'm a bit confused about some of its logic in places.  Is `shell-format` available outside of VS Code?  If not we might prefer to use something that is more universal (and can be added to CI).

**Review by Twey — APPROVED** [review]
I'd like to have a larger discussion about `shell-format` in general, but these changes seem perfectly cromulent :)

**Review by Twey — APPROVED** [review]
I have some questions, but no qualms :)

File: kubernetes/linera-validator/build_and_redeploy.sh (hunk 1 above)
Comment (andresilva91) [reply kubernetes/linera-validator/build_and_redeploy.sh:42]: It probably only keeps it in one line if there's a single command in it

Comment (Twey) [reply kubernetes/linera-validator/build_and_redeploy.sh:42]: Both are still one line, but on line 42 it demands that we add a space before the `;;`, while on line 41 it seems completely happy to leave the `;;` without a space.  The structure of the lines is more or less identical, so I'm confused as to what rule it used to decide :thinking:

Comment (andresilva91) [reply kubernetes/linera-validator/build_and_redeploy.sh:42]: Ah, I misunderstood what you said. Yeah, the space there but not on line 41 seems a bit arbitrary 🤔

**Comment by andresilva91** [comment]
> The changes look largely sensible to me, though I'm a bit confused about some of its logic in places.  Is `shell-format` available outside of VS Code?  If not we might prefer to use something that is more universal (and can be added to CI).

So it's a wrapper around `shfmt` https://github.com/mvdan/sh#shfmt
There's all sorts of integrations it seems, and `shfmt` can also be installed as a CLI tool

**Comment by andresilva91** [comment]
### Merge activity

* **Nov 6, 11:06 AM**: @andresilva91 started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/linera-io/linera-protocol/1205).

**Pull request merged**